crates/astria-sequencer-relayer/tests/blackbox/helper.rs

```diff
@@ -78,6 +101,44 @@ const PRIVATE_VALIDATOR_KEY: &str = r#"
 }
 "#;
 
+pub struct BlockGuard {
+    inner: oneshot::Receiver<()>,
+}
+
+impl BlockGuard {
+    pub async fn wait_until_satisfied(self) -> Result<(), tokio::sync::oneshot::error::RecvError> {
```

Comment:
Can you add a note that we should harmonize this and the abci mock guard so that we don't have to do the `.1.unwrap()` stuff when using this in a test?